Makefile

```diff
@@ -182,7 +188,10 @@ $(SCRIPTS_INSTALL_DIR) :
 
 install_common_scripts : $(SCRIPT_COMMON_INSTALL_DIR) $(COMMON_SCRIPTS_SRC)
 	$(CP) $(COMMON_SCRIPTS_SRC)/*.sh $(SCRIPT_COMMON_INSTALL_DIR)
-	
+
+install_core_scripts : $(SCRIPT_CORE_INSTALL_DIR) $(CORE_SCRIPTS_SRC)
+	$(RSYNC) --exclude *.rst $(CORE_SCRIPTS_SRC)/* $(SCRIPT_CORE_INSTALL_DIR)
```

Comment:
Rsync shouldn't be needed. If we are excluding a file, it means the directory we are going to install may have artifacts of a foglamp run (I am guessing). Typically any generated files shouldn't exist in a scripts/source directory (as it is readonly).

Reply:
> Why does this file appear within this directory?
you mean `.rst` file?